project/cmake/modules/LDGOLD.cmake

```diff
@@ -1,4 +1,8 @@
-option(ENABLE_LDGOLD "Use GNU gold linker" ON)
+if(CMAKE_BUILD_TYPE STREQUAL Release)
+  option(ENABLE_LDGOLD "Use GNU gold linker" ON)
+else()
+  option(ENABLE_LDGOLD "Use GNU gold linker" OFF)
+endif()
```

Comment:
well, this could be a bug in gold, but LE already had the same issue and it crashed on me as well with default ubuntu 16.04 toolchain. We simply cannot enable something that crashes.